packages/data-service/src/data-service.ts

```diff
@@ -159,7 +159,11 @@ export interface DataService {
     event: K,
     listener: DataServiceEventMap[K]
   ): this;
-  off?<K extends keyof DataServiceEventMap>(
```

Comment:
Couldn't find a reason for why this was an optional before, changed and added `removeListener` so that it's easier to use this with the plugin activate helpers